src/NuGet.Core/NuGet.Packaging.Core/INuspecCoreReader.cs

```diff
@@ -29,6 +29,11 @@ public interface INuspecCoreReader
         NuGetVersion GetMinClientVersion();
 
         /// <summary>
+        /// Gets zero or more package types from the .nuspec.
+        /// </summary>
+        IEnumerable<PackageType> GetPackageTypes();
```

Comment:
Yes, in the future we will allow multiple package types. Taking the quirks example, a tool could be both a `DotnetCliTool` and `Managed` package type. For now, we are limiting it to one.